gammapy/maps/core.py

```diff
@@ -87,6 +87,10 @@ def geom(self):
         """Map geometry as a `~gammapy.maps.Geom` object."""
         return self._geom
 
+    @property
+    def single_spatial_bin(self):
+        return self.geom.to_image().data_shape == (1, 1)
```

Comment:
I would avoid calling `to_image` in the `MapEvaluator.update` function because it can take time for no reason if e.g. you have a `RegionGeom`. If `is_single_spatial_bin` is a property on `Geom` then it can be always `True`.

Reply:
moved to IRFmap as a lazy property and renamed `has_single_spatial_bin `